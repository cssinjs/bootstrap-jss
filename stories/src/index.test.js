import React from 'react'
import renderer from 'react-test-renderer'

test('button', () => {

  const component = renderer.create(
    <button>hi</button>
  )

  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
