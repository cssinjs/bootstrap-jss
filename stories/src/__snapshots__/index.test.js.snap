// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button 1`] = `
<button>
  hi
</button>
`;
